org.signal.Signal-Beta.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "org.signal.Signal-Beta",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "signal",
+    "separate-locales": false,
+    "rename-desktop-file": "signal-desktop-beta.desktop",
+    "rename-icon": "signal-desktop-beta",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
+    ],
+    "modules": [
+        {
+            "name": "signal-desktop-beta",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar ox signal-desktop-beta_amd64.deb",
+                "tar xf data.tar.xz",
+                "mv 'opt/Signal Beta' opt/Signal-Beta",
+                "cp --recursive --preserve usr/* opt/* /app",
+                "rm -rf /app/share/icons/hicolor/1024x1024/ /app/Signal-Beta/chrome-sandbox",
+                "sed -i 's|Exec=.*|Exec=signal %U|' /app/share/applications/signal-desktop-beta.desktop",
+                "install signal.sh /app/bin/signal",
+                "install -Dm644 org.signal.Signal-Beta.appdata.xml /app/share/appdata/org.signal.Signal-Beta.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": ["x86_64"],
+                    "dest-filename": "signal-desktop-beta_amd64.deb",
+                    "url": "https://updates.signal.org/desktop/apt/pool/main/s/signal-desktop-beta/signal-desktop-beta_1.35.1-beta.3_amd64.deb",
+                    "sha256": "86a9b4bbb5839a0f00a6c08f2d5d71449d0724df8ebbd6edd1d106ea95dd2a9d",
+                    "x-checker-data": {
+                        "type": "debian-repo",
+                        "package-name": "signal-desktop-beta",
+                        "root": "https://updates.signal.org/desktop/apt",
+                        "dist": "xenial",
+                        "component": "main"
+                    }
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "signal.sh",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
+                        "zypak-wrapper /app/Signal-Beta/signal-desktop-beta --no-sandbox"
```

Comment:
It doesn't make sense to use `zypak` and `--no-sandbox` at the same time. I reccomend reporting issue to [zypak](https://github.com/refi64/zypak)

Reply:
Actually this is known issue and easily fixable: https://github.com/refi64/zypak#common-problems

After following above use `exec zypak-wrapper /app/Signal-Beta/signal-desktop-beta "$@"` as command to run.

Reply:
Very weird. The only reason I did use `--no-sandbox` was because even with that fix it wasn't working for me. Now it works 😅 À la "dans le doute, reboot" 🤷‍